### PR TITLE
fix: remove etcd configuration

### PR DESCRIPTION
**Pull request by gxthrj**
The ingress controller no dependencies with `etcd`, Related configuration and code deleted.

### CODE REVIEW

**Comment by gxthrj** [comment]
The test will supply in other PR later.

**Comment by Firstsawyou** [comment]
LGTM .

File: conf/init.go [hunk 1]
```diff
@@ -110,9 +102,9 @@ type syslog struct {
 	Host string
 }
 
-func GetEtcdAPI() client.KeysAPI{
-	return client.NewKeysAPI(etcdClient)
-}
+//func GetEtcdAPI() client.KeysAPI{
```
Comment (membphis) [inline conf/init.go:105]: If we do not need this code, we can delete them directly.

File: conf/init.go [hunk 2]
```diff
@@ -200,17 +192,17 @@ func ExceptNilErr(err error)  {
...
-		Endpoints: EtcdConfig.Addresses,
-		Transport: client.DefaultTransport,
-	}
-	if c, err := client.New(cfg); err != nil {
-		panic(fmt.Sprintf("failed to initialize etcd watcher. %s", err.Error()))
-	} else {
-		return c
-	}
-}
+//func NewEtcdClient() client.Client {
```
Comment (membphis) [inline conf/init.go:195]: ditto

**Review by moonming — APPROVED** [review]

File: conf/init.go (hunk 1 above)
Comment (gxthrj) [reply conf/init.go:105]: ok，I will Unified coding style  #42

File: conf/init.go (hunk 2 above)
Comment (gxthrj) [reply conf/init.go:195]: ok，I will Unified coding style  #42

**Pull request merged**